src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -58,7 +57,8 @@ internal JaegerExporter(JaegerExporterOptions options, TTransport clientTranspor
             this.memoryTransport = new InMemoryTransport(16000);
             this.memoryProtocol = this.protocolFactory.GetProtocol(this.memoryTransport);
 
-            this.Process = new Process(DefaultServiceName, options.ProcessTags);
+            string serviceName = (string) this.ParentProvider.GetDefaultResource().Attributes.FirstOrDefault().Value;
```

Comment:
we cannot assume the default will be containing a single attribute. Need to find the specific key we are after.